src/Settings.vala

```diff
@@ -78,4 +78,12 @@ namespace PantheonTerminal {
             base ("io.elementary.terminal.settings");
         }
     }
+
+    public class PrivacySettings : Granite.Services.Settings {
+        public bool remember_recent_files { get; set; }
+
+        public PrivacySettings () {
+            base ("org.gnome.desktop.privacy");
```

Comment:
This will crash if the setting schema is not installed (Maybe in a different OS). See main review comment :)

Reply:
Good point